pkgs/development/python-modules/pyperscan/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, buildPythonPackage
+, rustPlatform
+, pytestCheckHook
+, libiconv
+, vectorscan
+}:
+
+buildPythonPackage rec {
+  pname = "pyperscan";
+  version = "0.2.2";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "vlaci";
+    repo = "pyperscan";
+    rev = "v${version}";
+    hash = "sha256-ioNGEmWy+lEzazF1RzMFS06jYLNYll3QSlWAF0AoU7Y=";
+  };
+
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = ./Cargo.lock;
+  };
```

Comment:
I am not sure, what is the best practice in nixpkgs, I see examples 32 for `importCargoLock` but 81 for `fetchCargoTarball`